libs/ml/tests/unit/ops/activations/dropout.cpp

```diff
@@ -54,137 +70,114 @@ TYPED_TEST(DropoutTest, forward_test)
 
   op.Forward(vec_data, prediction);
 
-  // test correct values
-  EXPECT_TRUE(prediction.AllClose(gt, DataType{1e-5f}, DataType{1e-5f}));
+  // test correct fraction and sum
+  EXPECT_NEAR(zero_fraction(prediction), static_cast<double>(prob), 0.05);
```

Comment:
0.05 is presumably a slightly arbitrary value which is contingent on the size of the tensor? we could point this out with a comment